src/js/providers/tracks-mixin.js

```diff
@@ -624,32 +636,34 @@ define(['utils/underscore',
             this._cachedVTTCues[track._id] = {};
         }
         var cachedCues = this._cachedVTTCues[track._id];
-        var cacheKey;
+        var cacheKeyTime;
 
         switch (trackKind) {
             case 'captions':
+            case 'subtitles':
```

Comment:
Why do you need `subtitles` here? We don't need to cache subtitle cues because the `.vtt` files should only be parsed once.